packages/store/src/appSlice/index.ts

```diff
@@ -23,19 +24,99 @@ const getYesterday = () => {
   return today.toISOString();
 };
 
-export const initialState = {
+export type Repository = {
+  name: string;
+  version: string;
+};
+
+export type ReleaseVersion = {
+  release: string;
+  release_date: string;
+  repos: Repository[];
+};
+
+export type VersionRelease = {
+  [key: string]: ReleaseVersion | string;
+  supported_until: string;
+};
```

Comment:
I guess I should have been more explicit, but now with the type for the parsing in the `getLatestRelease` these are no longer aligned, so rather:
```suggestion
export type VersionRelease = Partial<{ [versionInfo: string]: Release }> & {
  release?: string;
  release_date?: string;
  repos?: Repository[];
  supported_until?: string;
};
```
here and have both `getLatestRelease` and `deductSaasState` share a common understanding of what a release can contain.